examples/tutorials/advanced/dataset_on_surface.py

```diff
@@ -0,0 +1,167 @@
+"""
+Draping a dataset on top of a topographic surface
+==================================================
+
+It can be visually appealing to "drape" a dataset over a topographic surface.
+This can be accomplished using the ``drapegrid`` parameter of
+:meth:`pygmt.Figure.grdview`.
+
+This tutorial consists of two examples:
+
+1. Draping a grid
+
+2. Draping an image
+"""
+
+# %%
+
+# Load the required packages
+import pandas as pd
+import pygmt
+import rasterio
+import xarray as xr
+
+# %%
+# 1. Drapping a grid
+# ------------------
+#
+# In the first example, the seafloor crustal age is plotted with color-coding
+# on top of the topographic map of an area of the Mid-Atlantic Ridge.
+
+# Define study area
+region_2d = [-50, 0, 36, 70]  # [lon_min, lon_max, lat_min, lat_max]
+
+# Create new Figure instance
+fig = pygmt.Figure()
+
+# Download elevation and crustal age grids for the study region with a
+# resolution of 3 arc-minutes and load them into xarray.DataArrays
```

Comment:
```suggestion
# Download elevation and crustal age grids for the study region with a
# resolution of 10 arc-minutes and load them into xarray.DataArrays
```

Reply:
Included in commit https://github.com/GenericMappingTools/pygmt/pull/3316/commits/965a29e97466585864280776e00ef0c9721f5599.